### DISCUSSION TITLE
downloading openfast

### DISCUSSION THREAD

**Discussion by jyotrimaya1993** [OpenFAST/openfast#1103] [Q&A]
being new to this software,i just want to know ,how to download openfast.

is matlab necessary to download openfast?

kindly help me ?

**Comment by jjonkman** [comment]
See: https://openfast.readthedocs.io/en/main/source/install/index.html.

MATLAB is not necessary, depending on what you want to do.

Best regards,

**Comment by jyotrimaya1993** [comment]
respected sir, for structural vibration control
For example:if in a floating offshore wind turbine model,if i am use
DAMPERs,then also MATLAB not required.

**Reply by jjonkman** [reply]
Dear @jyotrimaya1993,

The use of the structural control (StC) module of OpenFAST does not require MATLAB.  If you want to actively control StCs, you can do so either through MATLAB/Simulink or the the DISCON dynamic library control interface.

Best regards.

**Reply by jyotrimaya1993** [reply]
![Annotation 2022-05-10 153052](https://user-images.githubusercontent.com/104576729/167603528-5bfc65a1-4ce2-4495-ad8d-091380949803.jpg)
i have downloaded it,but why it is showing incompatible

**Reply by jjonkman** [reply]
Dear @jyotrimaya1993,

This "incompatible" error in Visual Studio would be triggered if you try to open up a Visual Studio project that is saved in a version that is different than the version of Visual Studio you have installed.  Similar questions were asked/answered here: 

https://github.com/OpenFAST/openfast/issues/354
https://github.com/OpenFAST/openfast/issues/328

Best regards,

**Comment by jyotrimaya1993** [comment]
sorry to say sir,but  problem are raising in openfast because in the document it is said to install as per cmake.......but in assets page it is showing cmake is not used......
that creat problem in installing basically for a beginner like me.....
![problem](https://user-images.githubusercontent.com/104576729/167997406-b8baa27e-eee5-4c1c-a4f2-ba3c4c75ca24.jpg)

**Reply by bjonkman** [reply]
There are multiple ways to install OpenFAST. On Windows, you can download the pre-compiled binaries, you can build the binaries using cmake, or you can build binaries using the Visual Studio files included in the git repo. You do not need to use binaries what were created with cmake. If you read that it must be installed with cmake, please send us the link where that is written so it can be fixed.
